rest/config.go

```diff
@@ -453,7 +453,7 @@ func readFromPath(path string, insecureSkipVerify bool) (rc io.ReadCloser, err e
 	messageFormat := "Loading content from [%s] ..."
 	if strings.HasPrefix(path, "http://") || strings.HasPrefix(path, "https://") {
 		base.InfofCtx(context.Background(), base.KeyAll, messageFormat, path)
-		client := base.GetHttpClient(insecureSkipVerify)
+		client := base.GetHttpClientForWebSocket(insecureSkipVerify)
```

Comment:
I'm not sure this is needed here (this http client is just used to fetch resources like remote config files).  The usage of GetHttpClient in db/active_replicator.go (for ISGR) looks like it should be updated, though.

Reply:
OK, reverted this one & changed the other.